DOCS-10105 Salesforce releases and migrations (#188)
mulesoft/docs-api-community-manager: modules/ROOT/pages/salesforce-releases-migrations.adoc

= Salesforce Releases and Migrations
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:imagesdir: ../assets/images/

> The Salesforce organization dedicated to your API community follows the standard https://trailhead.salesforce.com/en/content/learn/modules/sf_releases/sf_releases_start[Salesforce release process]. This automatically upgrades your organization to the latest Salesforce release version three times each year. As an organization administrator, you'll receive email notifications about the changes at least three months in advance, including any required steps for you to follow, guidance, and other resources.

Reviewer: present tense
```suggestion
The Salesforce organization dedicated to your API community follows the standard https://trailhead.salesforce.com/en/content/learn/modules/sf_releases/sf_releases_start[Salesforce release process]. This automatically upgrades your organization to the latest Salesforce release version three times each year. As an organization administrator, you receive email notifications about the changes at least three months in advance, including any required steps for you to follow, guidance, and other resources.
```
"steps, guidance, and resources" sounds odd to me: you? Also, "other resources" is vague.

> If any additional steps are required to prepare Anypoint API Community Manager, they will be provided from your MuleSoft customer success representative and on https://status.mulesoft.com/[trust.mulesoft.com]. Many Salesforce upgrades do not require any additional steps for API Community Manager.

Reviewer: passive, present
```suggestion
Additionally, if you must take steps to prepare Anypoint API Community Manager for the upgrade, your MuleSoft customer success representative provides them and you can also view them on https://status.mulesoft.com/[trust.mulesoft.com]. Many Salesforce upgrades do not require any additional steps for API Community Manager.
```
You could probably omit the second sentence, because the first specifies "if," which implies this?

> == Salesforce Organization Migrations

Reviewer: This is organizationally odd. The parent topic is about releases and migrations, and fully explains the release preparation process but not the migration process...which has become a subtopic:

= Releases and Migrations
== Migrations
== Getting Help

However, the migration text states that you typically experience no impact, but MuleSoft will let you know.
Then Getting Help repeats what the previous two topics say about Salesforce and MuleSoft help.

This topic could easily be just one text, titled something like Preparation for Salesforce Releases and Migrations.

> The Salesforce organization dedicated to your API community may sometimes need to move across instances within a region. These migrations usually have no impact on your API community. If a migration has any impact, you will be notified by your MuleSoft customer success representative and on https://status.mulesoft.com/[trust.mulesoft.com].

Reviewer: GLOBALLY replace "may" per style ref.; better still, omit modal verbs...also per style ref.
Ditto present tense.
```suggestion
The Salesforce organization dedicated to your API community sometimes moves across instances within a region. Although these migrations usually have no impact on your API community, your MuleSoft customer success representative and https://status.mulesoft.com/[trust.mulesoft.com] prepare you if there is impact.
```

> == Getting Help

Reviewer: We avoid gerund titles (-ing)
```suggestion
== How to Get Release or Migration Help
```
